### PR TITLE
wfuzz: edit page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/wfuzz.md [hunk 1]
```diff
@@ -3,18 +3,22 @@
...
+- Show [c]olorized output while only showing the declared response codes in the output:
 
 `wfuzz -c -w {{path/to/file}} --sc {{200,301,302}} {{http://example.com/FUZZ}}`
 
-- Use a custom header to fuzz subdomains while hiding specific response codes and word counts. Increase the threads to 100 and include the target ip/domain:
+- Use a custom [H]eader to fuzz subdomains while hiding specific response codes and word counts. Increase the [t]hreads to 100 and include the target ip/domain:
 
 `wfuzz -w {{path/to/file}} -H {{"Host: FUZZ.example.com"}} --hc {{301}} --hw {{222}} -t {{100}} {{example.com}}`
+
+- Brute force Basic Authentication using a list of usernames and passwords from files, [h]iding response [c]odes of unsuccessful attempts:
```
Comment (vitorhcl) [inline pages/common/wfuzz.md:22]:
```suggestion
- Brute force Basic Authentication using a list of usernames and passwords from files for each FUZ[z] keyword, [h]iding response [c]odes of unsuccessful attempts:
```

**Review by vitorhcl — APPROVED** [review]
LGTM, thank you! I have some suggestions

File: pages/common/wfuzz.md (hunk 1 above)
Comment (fazlearefin) [reply pages/common/wfuzz.md:22]: FUZ[z] does not fit into the style for multiple placeholders for `wfuzz`.

1st placeholder: FUZZ
2nd placeholder: FUZ2Z
3rd placeholder: FUZ3Z

So after the first FUZZ, the subsequent placeholders are FUZnZ where n >= 2

Here is a ref: https://www.hackingarticles.in/a-detailed-guide-on-wfuzz/
Search for **Double fuzzing** on the weblink above

Comment (vitorhcl) [reply pages/common/wfuzz.md:22]: It does fit on the style, because it refers to the z mnemonic refers to `-z`. However, I agree it can be improved.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**